### DISCUSSION TITLE
Is there a way to enable custom dubs?

### DISCUSSION THREAD

**Discussion by DenowRammy** [karashiiro/TextToTalk#73] [Ideas]
I'd be interested in a community project to put together custom dubs for the undubbed portions of XIV, would that be something this plugin could enable?

**Comment by karashiiro** [comment]
Probably not, what you’d want there is something with hooks into the actual cutscenes, so that you could uniquely identify a line of text from a particular cutscene. The same line from the same character might need to be spoken with a different tone in two different cutscenes, for example.

**Comment by karashiiro** [comment]
As a separate concern, I think that’d be scope creep? I can see the relationship between TTS and custom dubs, but it still seems a bit much. Something like that would benefit from being advertised as its own thing.
